### PR TITLE
update codecov task dependency

### PR DESCRIPTION
**Pull request by trajinthan**
#246

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -247,6 +247,7 @@ task build {
 }
 
 task codeCoverageReport(type: JacocoReport) {
+    dependsOn('build')
```
Comment (ThisaruGuruge) [inline build.gradle:250]: Shouldn't we update the other task dependencies as well?

i.e: `'http-ballerina-tests:codeCoverageReportBal'` should depends on the `http-ballerina:build` right?

Comment (trajinthan) [reply build.gradle:250]: yes, that's my fault, need to update `http-ballerina-tests:codeCoverageReportBal`dependency. I think i can remove the build dependency from here. am I right?

Comment (ThisaruGuruge) [reply build.gradle]: Yes, if we add dependencies for the other two tasks, we can remove the dependency here.

File: http-ballerina-tests/build.gradle [hunk 2]
```diff
@@ -239,6 +239,7 @@ task extractBallerinaClassFiles(type: Copy) {
 }
 
 task codeCoverageReportBal(type: JacocoReport) {
+    dependsOn('http-ballerina:build')
```
Comment (ThisaruGuruge) [inline http-ballerina-tests/build.gradle:242]:
```suggestion
    dependsOn(':http-ballerina:build')
```

File: http-native/build.gradle [hunk 3]
```diff
@@ -154,6 +154,7 @@ task extractJavaClassFiles(type: Copy) {
 }
 
 task codeCoverageReportJava(type: JacocoReport) {
+    dependsOn(':http-ballerina:build')
```
Comment (ThisaruGuruge) [inline http-native/build.gradle:157]: This should depend on `:http-native:build`.

Comment (trajinthan) [reply http-native/build.gradle:157]: yes, sorry i am in a hurry.

**Review by ThisaruGuruge — APPROVED** [review]

**Pull request merged**